--- host/python/CMakeLists.txt.orig	2022-07-19 05:15:57.000000000 +0800
+++ host/python/CMakeLists.txt	2022-11-01 18:44:41.000000000 +0800
@@ -20,7 +20,8 @@
 project(ubertooth-python)
 
 # Check for python installation
-find_package(Python3)
+set(Python3_FIND_UNVERSIONED_NAMES NEVER)
+find_package(PYTHON3)
 if(NOT Python3_FOUND)
 	return()
 else()

--- host/python/ubtbr/CMakeLists.txt.orig	2022-07-19 05:15:57.000000000 +0800
+++ host/python/ubtbr/CMakeLists.txt	2022-11-01 19:48:10.000000000 +0800
@@ -1,4 +1,5 @@
-find_program(PYTHON3 "python3")
+set(Python3_FIND_UNVERSIONED_NAMES NEVER)
+find_package(PYTHON3)
 set(SETUP_PY_IN	${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in)
 set(SETUP_PY    ${CMAKE_CURRENT_BINARY_DIR}/setup.py)
 set(DEPS        ${CMAKE_CURRENT_SOURCE_DIR}/ubtbr/__init__.py
@@ -9,12 +10,12 @@
 configure_file(${SETUP_PY_IN} ${SETUP_PY})
 
 add_custom_command(OUTPUT ${OUTPUT}/timestamp
-                   COMMAND ${PYTHON3} setup.py build
+                   COMMAND @PYTHON3@ setup.py build
                    COMMAND ${CMAKE_COMMAND} -E touch ${OUTPUT}/timestamp
                    DEPENDS ${DEPS})
 
 add_custom_target(ubtbr ALL DEPENDS ${OUTPUT}/timestamp)
-install(CODE "execute_process(COMMAND ${PYTHON3} ${SETUP_PY} build -b ${OUTPUT} install -f --prefix=${CMAKE_INSTALL_PREFIX})")
+install(CODE "execute_process(COMMAND @PYTHON3@ ${SETUP_PY} build -b ${OUTPUT} install -f --prefix=${CMAKE_INSTALL_PREFIX})")
 
 add_custom_target(ubertooth-btbr ALL DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/ubertooth-btbr)
 install(PROGRAMS ubertooth-btbr DESTINATION ${INSTALL_DEFAULT_BINDIR})
